x-pack/plugins/apm/public/components/app/service_logs/index.tsx

```diff
@@ -54,32 +51,6 @@ export function ServiceLogs() {
     [environment, kuery, serviceName, start, end]
   );
 
-  const noInfrastructureData = useMemo(() => {
-    return isEmpty(data?.containerIds) && isEmpty(data?.hostNames);
-  }, [data]);
-
-  if (status === FETCH_STATUS.LOADING) {
-    return (
-      <div style={{ textAlign: 'center' }}>
-        <EuiLoadingSpinner size="m" />
-      </div>
-    );
-  }
-
-  if (status === FETCH_STATUS.SUCCESS && noInfrastructureData) {
-    return (
-      <EuiEmptyPrompt
-        title={
-          <h2>
-            {i18n.translate('xpack.apm.serviceLogs.noInfrastructureMessage', {
-              defaultMessage: 'There are no log messages to display.',
-            })}
-          </h2>
-        }
-      />
-    );
-  }
-
```

Comment:
@smith For https://github.com/elastic/kibana/issues/142656 was it something like this you had in mind?

Reply:
I'm not sure why we even need our own empty state message. I assume the logs component itself has one.

Reply:
I don't recall the reason for us to have a separate no-data screen. Maybe we can load it faster, or perhaps we wanted it to look like our other views. But those are lesser concerns compared to actually being able to see the logs.